.github/workflows/build_and_test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         # The XCode version to use. If you want to update it please refer to this document:
         # https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-software
         # and set proper version.
-        XCODE_VERSION: 12.5
+        XCODE_VERSION: "13.0"
```

Comment:
Yaml was turning `13.0` into `13` during Xcode select, so had to force it to be a string instead of a number